src/Form/FormatConfigurationFormPdf.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\printable\Form\FormatConfigurationFormPdf
+ */
+
+namespace Drupal\printable\Form;
+
+use Drupal\printable\PrintableEntityManagerInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\pdf_api\PdfGeneratorPluginManager;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides shared configuration form for all printable formats.
+ */
+class FormatConfigurationFormPdf extends FormBase {
+
+  /**
+   * The printable entity manager.
+   *
+   * @var \Drupal\printable\PrintableEntityManagerInterface
+   */
+  protected $printableEntityManager;
+
+  /**
+   * Constructs a new form object.
+   *
+   * @param \Drupal\printable\PrintableEntityManagerInterface $printable_entity_manager
+   *   The printable entity manager.
+   */
+  public function __construct(PrintableEntityManagerInterface $printable_entity_manager) {
+    $this->printableEntityManager = $printable_entity_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('printable.entity_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'printable_configuration_pdf';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, $printable_format = NULL) {
+    $form['settings'] = array(
+      '#type' => 'fieldset',
+      '#title' => $this->t('PDF options'),
+    );
+    $form['settings']['print_pdf_pdf_tool'] = array(
+      '#type' => 'radios',
+      '#title' => $this->t('PDF generation tool'),
+      '#options' => array('mPDF' => 'mPDF', 'wkhtmltopdf' => 'wkhtmltopdf', 'TCPDF' => 'TCPDF'),
+      '#default_value' => (string)$this->config('printable.settings')->get('pdf_tool'),
```

Comment:
Why cast to string? The config schema already defines the type, I expect the config system to take care of providing the right type. If it is an integer, never mind PHP is forgiving.